recruitment/views.py

```diff
@@ -395,11 +395,18 @@ def decorator(*args, **kwargs):
 @remember_last_query_params('recruitment', [field for field in RecruitmentApplicationSearchForm().fields])
 def recruitment_period(request, year, pk, template_name='recruitment/recruitment_period.html'):
 	recruitment_period = get_object_or_404(RecruitmentPeriod, pk=pk)
+	eligible_roles = recruitment_period.eligible_roles
+
+	# We should be able to trust eligible_roles in the model,
+	# but in case something goes awry.
+	if eligible_roles < 1:
+		eligible_roles = 1
+	elif eligible_roles > 3:
```

Comment:
It is good to have the lower check but now that you have the solution of selecting priority in a dropdown I don't think we need the upper limit, it was more relevant before when each priority level would have its own column.

Reply:
Addressed in commit 81d678ac7627f58fd19e384831da62c55e01d54f!